src/Dapr.AzureFunctions.Extension/Dapr.AzureFunctions.Extension.csproj

```diff
@@ -10,19 +10,19 @@
     <Description>This extension adds bindings for Dapr</Description>
   </PropertyGroup>
 
-  <!-- Use C# 8.0 features -->
+  <!-- Use C# 9.0 features -->
   <PropertyGroup>
-    <LangVersion>8.0</LangVersion>
+    <LangVersion>9.0</LangVersion>
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
   <!-- Microsoft 1st party dependencies -->
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Http" Version="3.1.2" />
-    <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.2.7" />
+    <PackageReference Include="Microsoft.Extensions.Http" Version="7.0.0" />
```

Comment:
Consider using 6.0.0, since this is better aligned with .NET 6.0.

Reply:
I had to downgrade it back to the original version to get tests to pass. Yes I should try 6.0.0 -- all tests failed with 7.0.0. Took me so long to figure that out!